internal/servers/console_http_middleware_test.go

```
@@ -0,0 +1,241 @@
+			resp, err := http.Get(srv.URL)
+			Expect(err).NotTo(HaveOccurred())
+			resp.Body.Close()
+
+			metrics := metricsServer.Metrics()
+			Expect(metrics).To(osactesting.MatchLine(`console_websocket_connections_total\{.*status="error".*\} 1`))
+			Expect(metrics).NotTo(osactesting.MatchLine(`console_websocket_connection_duration_seconds_count`))
+		})
+
+		It("should count hijacked connection as success", func() {
+			handler := ConsoleMetrics(metricsServer.Registry(), http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+				conn, _, err := w.(http.Hijacker).Hijack()
```

golangci-lint (2.12.2)

[error] right hand must be only type assertion

(forcetypeassert)